ui/src/views/network/VpcTiersTab.vue

```diff
@@ -553,9 +553,9 @@ export default {
           if (this.publicLBExists && (idx === -1 || this.lbProviderMap.publicLb.vpc.indexOf(offering.service.map(svc => { return svc.provider[0].name })[idx]) === -1)) {
             filteredOfferings.push(offering)
           } else if (!this.publicLBExists && vpcLbServiceIndex > -1) {
-            const vpcLbServiceProvider = vpcLbServiceIndex === -1 ? undefined : this.resource.service[vpcLbServiceIndex].provider[0].name
+            const vpcLbServiceProviders = vpcLbServiceIndex === -1 ? undefined : this.resource.service[vpcLbServiceIndex].provider.map(provider => provider.name)
```

Comment:
a short explanation:
- the service has two providers: InternalLbVm and VpcVirtualRouter
- without this PR, the network offerings are listed only if (1) no LB; (2) provider is InternalLbVm
- with this PR, the network offerings are listed only if (1) no LB; (2)  provider is InternalLbVm; (3)  provider is VpcVirtualRouter